### PR TITLE
systemd-run: add page

### PR DESCRIPTION
**Pull request by cyqsimon**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** systemd-253

### CODE REVIEW

File: pages/linux/systemd-run.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# systemd-run
+
+> Run programs in transient units.
+> More information: <https://manned.org/systemd-run>.
+
+- Start a transient service:
+
+`sudo systemd-run {{command}} {{[args...]}}`
```
Comment (kbdharun) [inline pages/linux/systemd-run.md:8]:
```suggestion
`sudo systemd-run {{command}} {{argument1 argument2 ...}}`
```

File: pages/linux/systemd-run.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start a transient service under the service manager of the current user (no privileges):
+
+`systemd-run --user {{command}} {{[args...]}}`
```
Comment (kbdharun) [inline pages/linux/systemd-run.md:12]:
```suggestion
`systemd-run --user {{command}} {{argument1 argument2 ...}}`
```

File: pages/linux/systemd-run.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start a transient service with a custom unit name and description:
+
+`sudo systemd-run --unit={{name}} --description={{desc}} {{command}} {{[args...]}}`
```
Comment (kbdharun) [inline pages/linux/systemd-run.md:16]:
```suggestion
`sudo systemd-run --unit={{name}} --description={{string}} {{command}} {{argument1 argument2 ...}}`
```

File: pages/linux/systemd-run.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start a transient service that does not get cleaned up after it terminates with a custom environment variable:
+
+`sudo systemd-run --remain-after-exit --set-env={{name}}={{val}} {{command}} {{[args...]}}`
```
Comment (kbdharun) [inline pages/linux/systemd-run.md:20]:
```suggestion
`sudo systemd-run --remain-after-exit --set-env={{name}}={{value}} {{command}} {{argument1 argument2 ...}}`
```

File: pages/linux/systemd-run.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start a transient timer that periodically runs its transient service (see `man systemd.time` for calendar event format):
```
Comment (kbdharun) [inline pages/linux/systemd-run.md:22]:
```suggestion
- Start a transient timer that periodically runs its transient service:
```
I am not sure if this is required.

File: pages/linux/systemd-run.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo systemd-run --on-calendar={{calendar event}} {{command}} {{[args...]}}`
```
Comment (kbdharun) [inline pages/linux/systemd-run.md:24]:
```suggestion
`sudo systemd-run --on-calendar={{calendar_event}} {{command}} {{argument1 argument2 ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/systemd-run.md (hunk 5 above)
Comment (cyqsimon) [reply pages/linux/systemd-run.md:22]: I would like to keep it. If you take a loot at [`man systemd.time`](https://manned.org/systemd.time.7#head7), I think you can agree that the format is rather peculiar.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**